### ISSUE TITLE
kfctl build doesn't create all the istio_dex config

### ISSUE THREAD

**Issue by ryandawsonuk** [kubeflow/kfctl#175]
I do:
```
export CONFIG=https://raw.githubusercontent.com/kubeflow/manifests/master/kfdef/kfctl_istio_dex.yaml
kfctl build -f ${CONFIG} -V
```
The generated kustomize config should contain a folder titled 'istio' as my old build do:
![image](https://user-images.githubusercontent.com/22754674/72366186-adda3b00-36f1-11ea-878e-5b707e79d241.png)

But there is no such directory when I do a new build now. All the others are there, it's just this one that is missing. The result is that when I do a `kfctl apply` from the generated kfapp I get:
```
ERRO[0018] error evaluating kustomization manifest for istio Error absolute path error in '../kustomize/istio' : evalsymlink failure on '../kustomize/istio' : lstat /home/ryan/Documents/src/seldon-deploy/seldonctl/seldon-deploy-install/prerequisites-setup/tempresources/kf-install/kfapp/kustomize/istio: no such file or directory  filename="kustomize/kustomize.go:173"
Error: failed to apply:  (kubeflow.error): Code 500 with message: kfApp Apply failed for kustomize:  (kubeflow.error): Code 500 with message: error evaluating kustomization manifest for istio Error absolute path error in '../kustomize/istio' : evalsymlink failure on '../kustomize/istio' : lstat /home/ryan/Documents/src/seldon-deploy/seldonctl/seldon-deploy-install/prerequisites-setup/tempresources/kf-install/kfapp/kustomize/istio: no such file or directory
```

**Comment by yanniszark** [comment]
It seems that this is the offending commit https://github.com/kubeflow/manifests/commit/f9252596661bbfa7e4bf54c446797c6179b52abf#diff-22a62b4046d14c9056c87c46260349d7 introduced in PR https://github.com/kubeflow/manifests/pull/620.

cc @krishnadurai

**Comment by krishnadurai** [comment]
@ryandawsonuk Thanks for reporting this issue. I've added a fix here: kubeflow/manifests#704
